R/commonAnovaBayesian.R

```diff
@@ -504,7 +504,7 @@
 
   if (sum(!idxNotNan) > 1L) { # null model is always omitted, so 2 or more omitted indicates some models failed
     effectsTable$addFootnote(message = gettext("Some Bayes factors could not be calculated. Inclusion probabilities and Bayes factors are calculated while excluding these models. The results may be uninterpretable!"),
```

Comment:
Well imagine one model is excluded due to computational difficulties and its "true" posterior probability is 0.00001 ("true" as in, the correctly computed value). Now, the model-averaged results are not really affected by us excluding that model because it has a very small contribution. But if that one model had a posterior probability of 0.5, then by excluding it all model-averaged results are nonsense. The problem is, you can't tell what situation the user is in because we couldn't compute/ approximate the value... If you have a suggestion for an alternative warning I'm all ears!